using-paypal-for-ojs-and-ocs/en/your-real-paypal-account.md

```diff
@@ -2,7 +2,9 @@
 
 Once you’ve confirmed that payments from the pretend accounts are successful, you can now enable the PayPal Account.
 
-First,  you’ll need to change your need to change  your PayPal API app from Sandbox to Live. You will then need to generate the Live API credentials. 
+First,  you’ll need to change your need to change your PayPal API app from Sandbox to Live. You will then need to generate the Live API credentials. To get Live API Credentials, go to Tools > Business Setup > Offer PayPal checkout on your website. Click **Set Up Online Payments** followed up **Get Your API Credentials**
```

Comment:
Looks like some text is duplicated here in the first sentence

Reply:
Just made an additional change to correct this.

Reply:
Sorry for not being more specific - I was referring to the start of the paragraph (which was an issue before your PR): "you’ll need to change your need to change your"

Reply:
Thanks for clarifying and I'm sorry for missing that. I've removed the duplicate in the sentence.